### ISSUE TITLE
Feature request: display "dirty" / "uncommitted changes" icon in the status bar

### ISSUE THREAD

**Issue by r1cc4rdo** [denolehov/obsidian-git#243]
**Tl;dr**: how feasible would it be to display the presence of unpushed changes in the status bar?

I arrived here from issue #13, searching for a way to make sure all changes are always saved. For obsidian git to trigger X minutes after a change, it needs a way to monitor if changes are present. If that's indeed the case, I wish the state of the repo was shown in the status bar: this would allow me to know at a glance if it's safe to close the application, and allow me to either wait or invoke the backup command manually if needed.

Right now, I always see a checkmark in the status bar, regardless of the state of the repo: the only time I see it change is when actively running command, _i.e._ committing or pushing in the background.

Platform: Mac OS 12.0.1
Obsidian version: 0.14.6
Obsidian git: 1.25.3
